smp/include/smp/smp.h

```diff
@@ -84,6 +84,17 @@ struct smp_streamer {
  */
 int smp_process_request_packet(struct smp_streamer *streamer, void *req);
 
+#ifdef CONFIG_MCUMGR_SMP_NOTIFY_CALLBACK
```

Comment:
Is it common for mcumgr to take dependency on Zephyr's configs?

Reply:
Actually that's a good point, I hadn't realized until I split up the PR that it was in a different repo. I think there's another header file in zephyr that I can use and will work just as well.

Reply:
👍